pyke/kepfunc.py

```diff
@@ -775,3 +777,116 @@ def inv_normal_cummulative_function(p):
     r = q * q
     return ((((((a[0] * r + a[1]) * r + a[2]) * r + a[3]) * r + a[4]) * r + a[5]) * q /
             (((((b[0] * r + b[1]) * r + b[2]) * r+ b[3]) * r + b[4]) * r + 1))
+
+def read_and_interpolate_prf(prfdir, module, output, column, row, xdim, ydim,
+                             verbose=False, logfile='kepprf.log'):
+    """
+    Read PRF file and prepare the data to be used for evaluating PRF.
+
+    Parameters
+    ----------
+    prfdir : str
+        The full or relative directory path to a folder containing the Kepler
+        PSF calibration. Calibration files can be downloaded from the Kepler
+        focal plane characteristics page at the MAST.
+    module : str
+        The 'MODULE' keyword from TPF file.
+    output : str
+        The 'OUTPUT' keyword from TPF file.
+    column : int
+        The '1CRV5P' keyword from TPF[1] file.
+    row : int
+        The '2CRV5P' keyword from TPF[1] file.
+    xdim : int
+        The first part of the 'TDIM5' keyword from TPF[1] file.
+    ydim : int
+        The second part of the 'TDIM5' keyword from TPF[1] file.
+    verbose : boolean
+        Print informative messages and warnings to the shell and logfile?
+    logfile : string
+        Name of the logfile containing error and warning messages.
+
+    Returns
+    -------
+    splineInterpolation
+        You can get PRF at given position: 
+        kepfunc.PRF2DET([flux], [x], [y], DATx, DATy, 1.0, 1.0, 0.0, splineInterpolation)
+    DATx : numpy.array
+        X-axis coordiantes of pixels for given TPF
+    DATy : numpy.array
+        Y-axis coordinates of pixels for given TPF
+    prf : numpy.array
+        PRF interpolated to given position on the camera
+    PRFx : numpy.array
+        X-axis coordinates of prf values
+    PRFy : numpy.array
+        Y-axis coordinates of prf values
+    PRFx0 : int
+    PRFy0 : int
+    cdelt1p : numpy.array 
+        CDELT1P values from 5 HDUs of PRF file.
+    cdelt2p : numpy.array
+        CDELT2P values from 5 HDUs of PRF file.
+    prfDimX : int
+        size of PRFx
+    prfDimY : int
+        size of PRFy
+
+    """
+    n_hdu = 5
+    minimum_prf_weight = 1.e-6
+
+    # determine suitable PRF calibration file
+    if int(module) < 10:
+        prefix = 'kplr0'
+    else:
+        prefix = 'kplr'
+    prfglob = os.path.join(prfdir, prefix + module + '.' + output + '*_prf.fits')
```

Comment:
Let's make sure we add ``import os`` and ``import glob`` at the top of the ``kepfunc.py`` module.